ouroboros-consensus-cardano-test/src/Test/ThreadNet/TxGen/Cardano.hs

```diff
@@ -11,6 +11,7 @@ module Test.ThreadNet.TxGen.Cardano (
     CardanoTxGenExtra (..),
   ) where
 
+import           Cardano.Ledger.Val ((<->))
```

Comment:
Can you move this down to where it used to be? Even though it doesn't match the `Shelley.Spec.Ledger` namespace, it is defined in the same package, and we *try* (but often fail) to group imports per IOHK package (not for general hackage packages).